lib/pure/json.nim

```diff
@@ -908,15 +908,14 @@ proc parseJson*(s: Stream, filename: string = ""; rawIntegers = false, rawFloats
 
 when defined(js):
   from std/math import `mod`
-  type
-    JSObject = object
+  import std/jsffi
```

Comment:
jsffi defines as:
`JsObject* = ref object of JsRoot`

but json was defining it as
`type JSObject = object`

can we assess consequences of this change?

Reply:
`tjson.nim` support JS backend already. I will look into it a bit more.

Reply:
I find a pre-existing bug

```nim
import std/json

echo parseJson("")
```

undefined:1
SyntaxError: Unexpected end of JSON input

Reply:
But I don't think they have much difference, they are only used to interact with js(as return types or parameters).